aws-cpp-sdk-core/source/utils/crypto/openssl/CryptoImpl.cpp

```diff
@@ -35,16 +35,19 @@ namespace Aws
         {
             namespace OpenSSL
             {
+#if OPENSSL_VERSION_NUMBER < 0x10100003L
                 static const char* OPENSSL_INTERNALS_TAG = "OpenSSLCallbackState";
                 static std::mutex* locks(nullptr);
 
+#endif
                 GetTheLights getTheLights;
 
                 void init_static_state()
                 {
                     ERR_load_CRYPTO_strings();
                     OPENSSL_add_all_algorithms_noconf();
 
+#if OPENSSL_VERSION_NUMBER < 0x10100003L
```

Comment:
Why is the locking setup removed here?  Is there some documentation somewhere on why it's no longer necessary.  I can't find anything about it being deprecated.